server/routes/users.js

```diff
@@ -85,9 +75,14 @@ export default (storage, scriptManager) => {
   /*
    * Get a single user.
    */
-  api.get('/:id', verifyUserAccess('read:user', scriptManager), (req, res, next) => {
```

Comment:
Why was this changed?

Reply:
It probably should not be... I will look at this.

Reply:
Added back in